### ISSUE TITLE
Part of the cookie is invalid

### ISSUE THREAD

**Issue by frankhommers** [AngleSharp/AngleSharp#732]
These steps work with a browser, but not with AngleSharp like this:

```csharp
IConfiguration config = Configuration.Default.WithDefaultLoader().WithCookies();
IBrowsingContext browsingContext = BrowsingContext.New(config);
browsingContext.OpenAsync("http://192.168.111.1/cgi-bin/luci").Wait();
(browsingContext.Active.QuerySelector("input[name = 'luci_username']") as IHtmlInputElement).Value = userName;
(browsingContext.Active.QuerySelector("input[name = 'luci_password']") as IHtmlInputElement).Value = password;
(browsingContext.Active.QuerySelector("form") as IHtmlFormElement).SubmitAsync().Wait();
```

Then I get this exception:

```text
Exception has occurred: CLR/System.AggregateException
An unhandled exception of type 'System.AggregateException' occurred in System.Private.CoreLib.dll: 'One or more errors occurred.'
 Inner exceptions found, see $exception in variables window for more details.
 Innermost exception 	 System.Net.CookieException : The 'Path'='/cgi-bin/luci/' part of the cookie is invalid.
   at System.Net.Cookie.VerifySetDefaults(CookieVariant variant, Uri uri, Boolean isLocalDomain, String localDomain, Boolean setDefault, Boolean shouldThrow)
   at System.Net.CookieContainer.CookieCutter(Uri uri, String headerName, String setCookieHeader, Boolean isThrow)
```

Any suggestions?

**Comment by FlorianRappl** [comment]
Quick fix: Try rolling out your own cookie container which can handle these cookies.

Question here is - can you try to run your code on AngleSharp v0.10 (`devel` branch)? Does the issue still persist there? If the issue persists it needs to be fixed in v0.10.

Thanks for the report!

**Comment by FlorianRappl** [comment]
Closing due to inactivity.
